app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -261,7 +260,9 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
       validStakepools[poolHash] ? validStakepools[poolHash].url : null
 
     const poolMetaUrls = distinct(
-      [...delegations, ...rewards].map(({poolHash}) => extractUrl(poolHash))
+      [...delegations, ...rewards]
+        .filter(({name}) => !name)
+        .map((historyEntry) => extractUrl(historyEntry.poolHash))
```

Comment:
Isn't object destructing more clear and shorter syntax?

Reply:
It is, my bad I changed it during the refactor and forgot to change it back. I ll fix it.